src/routes/settings/settings.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "../../components"
+
+export const Settings: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightGray items-center gap-8">
+      <div className="flex flex-col items-center gap-8 my-[60px]">
```

Comment:
Add bg-lightGray here. 
gap-8 is not needed if you restore button.tsx back to how it was.
my-[60px] adds a margin of 60 on top and on bottom, which isn't what we want. 
Use pb for padding bottom, the value would be 11 (since we restored button.tsx back to have py-4, you can subtract  16px from 60px which is 44px, which corresponds to 11 REM).

The whole line should look like this:
<div className="flex flex-col items-center pb-11 bg-lightGray">
And would look identical to how it was before, if my previous comments were implemented.